server/parsec/components/account.py

```diff
@@ -24,6 +29,23 @@ async def test_new_account(self) -> str:
         email = f"{uuid4().hex[:4]}@invalid.com"
 
         # create account
-        assert await self.create(email) is None
+        await self.create_token(email)
 
         return email
+
+    @api
+    async def api_account_send_email_validation_token(
+        self,
+        client_ctx: AnonymousAccountClientContext,
+        req: anonymous_account_cmds.latest.account_send_email_validation_token.Req,
+    ) -> anonymous_account_cmds.latest.account_send_email_validation_token.Rep:
+        outcome = await self.create_token(req.email)
+        match outcome:
+            case CreateAccountBadOutcome.INVALID_EMAIL:
+                return anonymous_account_cmds.latest.account_send_email_validation_token.RepInvalidEmail()
+            case CreateAccountBadOutcome.ACCOUNT_ALREADY_EXISTS:
+                # Respond OK without sending token to prevent creating
+                return anonymous_account_cmds.latest.account_send_email_validation_token.RepOk()
+            case EmailValidationToken():  # as token:
+                # TODO send email with token
```

Comment:
This TODO is tracked in #9956 as part of the added route. Will you be adding it to this PR?

Reply:
As it's its own can of worms, and we want to keep PRs small I'd rather do it separately